Assets/MixedRealityToolkit/StandardAssets/Shaders/MRTK_Wireframe.shader

```diff
@@ -115,7 +115,8 @@ Shader "Mixed Reality Toolkit/Wireframe"
                    o.viewPos = i[idx].viewPos;
                    o.inverseW = 1.0 / o.viewPos.w;
                    o.dist = distScale[idx] * o.viewPos.w * wireScale;
-                   UNITY_TRANSFER_VERTEX_OUTPUT_STEREO(i[idx], o);
+				   UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO(o);
+				   UNITY_TRANSFER_VERTEX_OUTPUT_STEREO(i[idx], o);
```

Comment:
```suggestion
                   UNITY_TRANSFER_VERTEX_OUTPUT_STEREO(i[idx], o);
```

Reply:
I would recommend moving UNITY_INITIALIZE_VERTEX_OUTPUT_STEREO to just below line 110 (g2f o;) so that the vertex get's initialized just once (and not every loop). I'm still not clear why we would need to initialize the 'o' since it looks like are members are getting set? If you look at this [built in Unity geometry shader](https://github.com/TwoTailsGames/Unity-Built-in-Shaders/blob/master/DefaultResourcesExtra/VR/Shaders/SpatialMappingWireframe.shader) we could try follow that macro placement to see if the error presists on Android.

Reply:
Maybe this is the macro we need: UNITY_SETUP_STEREO_EYE_INDEX_POST_VERTEX https://docs.unity3d.com/Manual//Android-SinglePassStereoRendering.html I can't find much documentation on it, but it is called out in that android specific doc.